content/pt-br/docs/reference/glossary/wg.md

```diff
@@ -0,0 +1,19 @@
+---
+title: WG (Grupo de Trabalho)
+id: wg
+date: 2018-04-12
+full_link: https://github.com/kubernetes/community/blob/master/sig-list.md#master-working-group-list
+short_description: >
+  Facilita a discussão e/ou implementação de um projeto de curta duração, curto ou dissociado para um comitê, SIG ou cross-SIG.
+
+aka: 
+tags:
+- community
+---
+ Facilita a discussão e/ou implementação de um projeto de curta duração, curto ou dissociado para um comitê, {{< glossary_tooltip text="SIG" term_id="sig" >}} ou cross-SIG.
```

Comment:
Is “cross-SIG” idiomatic Portguesese?

Reply:
Suggestion: instead of "cross-SIG", you could use "envolvendo um ou mais SIGs (grupos de interesse especial)."

I also think that "projeto de curta duração" is redundant with "curto". "Narrow", in this context, is more about scope. So you could say something like "de escopo limitado" or "pontual", in lieu of "curto".

Reply:
The suggestion from @stormqueen1990 is great, I agree with this: **"envolvendo um ou mais SIGs (grupos de interesse especial)."**
@MrErlison can you apply this suggestion, please?
Thank you guys

Reply:
Thank you. Sorry for not adding you as co-authors, I do not know how to do it.